README.md

```diff
@@ -1,2 +1,28 @@
-# miden-formatting
-Pretty-printing and formatting utilities for Miden crates
+# `miden-formatting`
+
```

Comment:
I usually try to add some badges right after the title (e.g., like [here](https://github.com/0xPolygonMiden/miden-vm/blob/main/README.md?plain=1#L3-L6)).